src/components/widgets/status/StatusTab.vue

```diff
@@ -341,6 +341,24 @@ export default class StatusTab extends Mixins(StateMixin, FilesMixin) {
     return this.$store.state.printer.printer.current_file.filament_total || 0
   }
 
+  /**
+   * Total filament weight according to the current file / slicer.
+   */
+  get filament_weight_total () {
```

Comment:
could you remove this extra function, and put the single line between 354 and 355 below?